src/locales/ar/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,623 @@
+"✔️ Wordpress,\n"
+"✔️ Notepad++,\n"
+"✔️ Ubuntu\n"
+"and thousands of common software started out as Open-source software? 👇🏼"
+msgstr ""
+" 👩🏽‍🏫  هل تعرف أن  ...\n"
+" ✔️ Wordpress,\n"
+"✔️ Notepad++,\n"
+"✔️ Ubuntu\n"
+"وآلاف البرامج الشائعة بدأت كبرامج مفتوحة المصدر؟ 🏼👇 "
+
+#: services/messenger.py:261
+msgid ""
+"😎 Worry not!\n"
+"\n"
+"Version control allows you to manage changes to files over time ⏱️."
+msgstr ""
+"😎لا تقلق!\n"
+" يسمح لك التحكم في الإصدار بإدارة التغييرات على الملفات بمرور الوقت  ️.⏱ "
+
+#: services/messenger.py:271
+msgid ""
+"You can use version control to version code, binary files, and digital "
+"assets 🗄️."
+msgstr ""
+" يمكنك استخدام ادوات  التحكم في الإصدار لتعيين إصدار  ملفات شفرة المصدر "
+"وملفات الصيغة الثنائية (لغة الألة) بالإضافة إلى الأصول الرقمية  .🗄️ "
+
+#: services/messenger.py:280
+msgid ""
+"This includes version control software, version control systems, or "
+"version control tools 🧰."
+msgstr ""
+" يتضمن ذلك برامج التحكم في الإصدار أو أنظمة التحكم في الإصدار أو أدوات "
+"التحكم في الإصدار  .🧰 "
+
+#: services/messenger.py:289
+msgid "Version control is a component of software configuration management 🖥️."
+msgstr " التحكم في الإصدار هو أحد مكونات إدارة تكوين البرامج  .🖥️ "
+
+#: services/messenger.py:300
+msgid ""
+"😎 Now that you understand what Version control is, let's explore another "
+"important topic."
+msgstr ""
+" 😎  الآن بعد أن فهمت ماهية التحكم في الإصدار ، دعنا نستكشف موضوعًا آخر "
+"مهمًا. "
+
+#: services/messenger.py:311
+msgid "What is Git❔"
+msgstr " ما هو  Git  ؟ "
+
+#: services/messenger.py:313
+msgid "What is GitHub❔"
+msgstr " ما هو  GitHub  ؟ "
+
+#: services/messenger.py:316
+msgid "What do you want to start with⁉️ 👇🏼"
+msgstr " بماذا تريد أن تبدأ!؟  🏼👇 "
+
+#: services/messenger.py:326
+msgid ""
+"GitHub is a code hosting platform for version control and collaboration. "
+"It lets you and others work together on projects from anywhere."
+msgstr ""
+" GitHup  هو عبارة عن منصة لاستضافة رموز البرمجيات للتحكم في الإصدار "
+"والتعاون.  حيث يتيح لك ولغيرك العمل معًا في المشروعات من أي مكان. "
+
+#: services/messenger.py:339
+msgid "Official Website"
+msgstr "الموقع الرسمي"
+
+#: services/messenger.py:344
+msgid "GitHub Tutorial"
+msgstr "  دروس تعليمية لـ  GitHub "
+
+#: services/messenger.py:349
+msgid "👩‍💻 Make a PR"
+msgstr " 👩‍💻 قدم طلب سحب"
+
+#: services/messenger.py:354
+msgid "Discover GitHub"
+msgstr " تعرف على  GitHub "
+
+#: services/messenger.py:356
+msgid "Discover GitHub official website, or follow a beginner tutorial"
+msgstr " اكتشف موقع  GitHub  الرسمي ، أو اتبع الدروس التعليمية للمبتدئين "
+
+#: services/messenger.py:370 services/messenger.py:711
+msgid "Good question 👌🏽"
+msgstr " سؤال جيد  👌🏽 "
+
+#: services/messenger.py:374 services/messenger.py:715
+msgid ""
+"Git is a type of version control system (VCS) that makes it easier to "
+"track changes to files. "
+msgstr ""
+" Git  هو نوع من أنواع نظم التحكم في الإصدارات  (VCS)  يُسهل عليك تعقب "
```

Comment:
Just remove the word VCS! doesn't really make sense here

Reply:
Done